src/redux/reducers/index.js

```diff
@@ -1,8 +1,8 @@
 import { combineReducers } from 'redux';
-import simpleReducer from 'redux/reducers/simpleReducer';
+import fetchEpisodes from 'redux/reducers/fetchEpisodes';
```

Comment:
names of reducers should be more about data which they refer to, so in `episodes.js` reducers you should take care of actions like `FETCH_EPISODES_REQUEST`, ``FETCH_EPISODES_FAILURE`, `FETCH_EPISODES_SUCCESS` and maybe more in the future